js/common/lib/inference-session.ts

```diff
@@ -223,6 +223,15 @@ export declare namespace InferenceSession {
   export interface CudaExecutionProviderOption extends ExecutionProviderOption {
     readonly name: 'cuda';
     deviceId?: number;
+    gpuMemLimit?: bigint;
```

Comment:
from the API, we want to keep it easier for users to use. JavaScript Number (double) can hold exact integer values within range [-2^53, 2^53), which is big enough for specifying gpuMemLimit.

please change the type back to `number` or `number | bigint` if you want to support both.

Reply:
Oh sorry. Somehow I didn't realize...